grin/test/AbstractInterpretation/ExtendedSyntax/CreatedBySpec.hs

```diff
@@ -112,10 +112,12 @@ spec = do
             f x =
               z1 <- pure 0
               case x of
-                0 -> x0 <- pure (CInt z1)
-                     pure x0
-                1 -> x1 <- pure (CBool z1)
-                     pure x1
+                0@_1 ->
```

Comment:
OMG :) This is soo confusing, first I parsed it as a numeral literal... :D

Reply:
You should have called these as "a1" and so on...

Reply:
I added spaces around the `@`.